sdk/deviceupdate/azure-deviceupdate/azure/deviceupdate/__init__.py

```diff
@@ -0,0 +1,18 @@
+# coding=utf-8
```

Comment:
Looking at your [config file](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/deviceupdate/data-plane/readme.python.md), if you remove `no-namespace-folder: true` from the config, you should have your `output-folder` just set to `$(python-sdks-folder)/sdk/deviceupdate/azure-iot-deviceupdate`. The namespace folders we generate will automatically fill in the rest

Reply:
Do we want to do that? I can try to make that change.